package io.github.orioneee

import io.github.orioneee.domain.exceptions.AxerException

internal actual fun notifyAboutException(exception: AxerException) {}